src/main/java/com/sap/sgs/phosphor/fosstars/data/IsEclipse.java

```diff
@@ -12,20 +12,23 @@
 public class IsEclipse implements DataProvider {
 
   /**
-   * A GitHub organization.
+   * An organization or username on GitHub.
    */
   private final String where;
 
   /**
-   * @param where A GitHub organization.
+   * Initializes a data provider.
+   *
+   * @param where An organization or username on GitHub.
    */
   public IsEclipse(String where) {
     this.where = where;
   }
 
   @Override
   public IsEclipse update(ValueSet values) {
-    System.out.println("[+] Figuring out if the project belongs to the Eclipse Software Foundation ...");
+    System.out.println(
```

Comment:
You want these sysouts?

Reply:
Yes, this output is used in the demo tool. It's better to use a logger but it's out of scope here.

Reply:
https://github.com/SAP/fosstars-rating-core/issues/35 yes I did.